data/hai/hai reveal 4 middle.txt

```diff
@@ -104,16 +104,16 @@ mission "Hai Reveal [A02] Symbol Identification"
 				goto questions
 			label cadmium
 			`	A Hai in a medical uniform explains, "The victims each seem to have been exposed to airborne particulates containing cadmium which must be circulating through their ships ventilation. I should also clarify that it's not strictly 'poisoning'. The level of exposure is quite low, but enough that testing shortly after being rescued does show clearly elevated levels in many who were taken.`
-			`	"No known technology of human or Hai origin exhausts anything which could create this effect. There are some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding, and their dispenser canisters have residual cadmium dust in them as residue, but that's the only thing we even bring onto a ship that uses it, and those are self-contained for disposal and the molecular bonding doesn't match anyway. There are some materials involved have properties unlike anything either of us make, which suggests this is coming from another race's technology."`
+			`	"No known technology of human or Hai origin exhausts anything which could create this effect. There are some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding. Their dispenser canisters have a residue of residual cadmium dust in them, but those are the only thing we even bring onto a ship that uses it and are self-contained for disposal. The molecular bonding doesn't match anyway. There are some materials involved with properties unlike anything either of us make, which suggests this is coming from another race's technology."`
 			`	They share a glance with Teeneep, "We have our suspicions but they're not conclusive yet."`
 			branch questions
 				not "00 Hai Leaks Response 4B: declined"
-			`	"We did; however, get a hold of the scan results from the Geocoris you rescued. Interestingly, the self-repair systems it has appear to use a technology which could be responsible for something like this, and we suspect they came from the same place."`
+			`	"We did, however, get a hold of the scan results from the Geocoris you rescued. Interestingly, the self-repair systems it has appear to use a technology which could be responsible for something like this, and we suspect they came from the same place."`
```

Comment:
I'm not sure semicolons are used this way.

Reply:
Iirc, both are correct, but it's a matter of regional English. The way Grey uses it being considered (out)dated in US English, and elsewhere it may still be considered the sole proper form.